.github/workflows/remote-build-ios.yml

```
@@ -40,7 +38,7 @@ jobs:
       - name: RNEF Remote Build - iOS
         uses: callstackincubator/ios@f8aeffabe13be32e9aa69b86726de67bff5a694e
         env:
-          PROJECT_ROOT_PATH: ${{ matrix.project_root }}
+          IS_HYBRID_APP: ${{ matrix.is_hybrid_build && 'true' || 'false' }}
```

NAB but if you put `is_hybrid_build: false` in the standalone matrix variant then I think you could just have this:

```suggestion
          IS_HYBRID_APP: ${{ matrix.is_hybrid_build }}
```